test-node.bash

```diff
@@ -33,8 +36,9 @@ detach=false
 blockscout=true
 tokenbridge=true
 redundantsequencers=0
+dev_build=false
 batchposters=1
-devprivkey=e887f7d17d07cc7b8004053fb8826f6657084e88904bb61590e498ca04704cf2
+devprivkey=b6b15c8cb491557369f3c7d2c287b053eb229daa9c22138887752191c9520659
```

Comment:
I found one reference online to the current devprivkey: https://www.wenwoha.com/blog_detail-2354.html
also, current devprivkey is used in config.ts, but I think we can remove it from there and use empty-init instead (funds are sent from l1, no need to have funds on initialization of l2)